nixos/tests/taskserver.nix

```diff
@@ -176,7 +176,7 @@ in {
 
     sub checkClientCert ($) {
       my $user = $_[0];
-      my $cmd = "gnutls-cli".
+      my $cmd = "gnutls-cli -d 3".
```

Comment:
Could you please add a comment on why this is needed?

Reply:
Done.